docs/swagger/spec.yml

```diff
@@ -2331,7 +2447,73 @@ definitions:
       maxModifiedDatetime:
         type: string
         format: datetime
+  UploadCreate:
+    type: object
+    properties:
+      uploadStatus:
+        type: string
+        description: Status of upload
+        enum:
+          - CREATED
+          - UPLOADING
+          - UPLOADED
+          - QUEUED
+          - PROCESSING
+          - COMPLETE
+      files:
+        type: array
+        items:
+          type: string
+      uploadType:
+        type: string
+        description: Source of files and uploads
+        enum:
+          - DROPBOX
+          - S3
+          - LOCAL
+      fileType:
+        type: string
+        description: type of data being uploaded, limited to two options right now
+        enum:
+          - GEOTIFF
+          - GEOTIFF_WITH_METADATA
+      datasource:
+        type: string
+        description: datasource of tiffs being uploaded
+        format: uuid
 
+  Upload:
+    allOf:
+    - $ref: '#/definitions/BaseModel'
+    - $ref: '#/definitions/UserTrackingMixin'
+    - $ref: '#/definitions/UploadCreate'
+    - type: object
+      properties:
```

Comment:
In our spec, some items have `properties` as a direct child, and others have it as a child of `allOf`. For example, see `ProjectPaginated`:

```yaml
ProjectPaginated:
  allOf:
    - $ref: '#/definitions/PaginatedResponse'
  properties:
    results:
      type: array
      items:
        $ref: '#/definitions/Project'
```

But there are others which follow the format used here. Do you know which one is correct? And if it is a stylistic matter, we should probably pick one and apply it uniformly. Might be a good idea to make a card for this.